purchase_mass_mail/models/mail_compose_message.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2023 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+from odoo import models
+
+
+class MailComposeMessage(models.TransientModel):
```

Comment:
This is a wizard, so the standard is to put the file inside the `wizards/` folder. Could you move it please?